src/features/purchases/screen/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { SafeAreaView, SectionList, Text, View } from 'react-native';
+
+import { translate } from 'localization/hooks';
+
+import { useGetOrders } from 'network/queries/order-queries';
+
+import OrderItem from './orderItem';
+import styles from './styles';
+
+const PurchasesScreen: React.FunctionComponent = () => {
+  const { data } = useGetOrders();
+  const orders = data?.data || [];
+
+  const sections =
+    orders.length > 0
```

Comment:
Cool thing about React Query is you can transform data, there is a `select` function in the `useQuery` options where you can create the section's object (also checking if you have orders to do ti), and then return {data, section, ...rest} and use it in the component without having to validate if orders length.